Eze

delucaezequiel opened the pull request.

Commit:
Merge Conflicts

Commit:
Controlador:

ABM Productos
ABM Clientes

Responsables se agrego el actualizar en base que faltaba

MacarenaBS reviewed (changes requested):
Eze, 
En el pull request de ahora esta cambiado el tipo de codigo de cliente de int a String, podes volver a ponerlo  como  estaba antes? porque con  los cambios que hice yo, el newID se genera como numerito y desde el dao, asi que puede volver a estar como antes. 

En el controlador fijate que el crearUsuario lo estas  sacando del DAO en vez de del admin, igual eso lo cambio yo sino no hay drama pero volver a cambiar todos los  strings  por int me va a infartar. Dejemos todo como esta en la bd. 
Otra cosa, que hacemos, que el controlador tire los cuadros de mensaje de error o las ventanas? porque ahora las ventanas tiran el mensaje de error, si mergeo este controlador van a saltar como 20 ventanas jaja, pero bueno, eso lo podemos ver...

Subo  mis cambios del newID() y las otras pelotudeces y volvé a hacer el pull request con tu controlador pero con los codigos con int plz

Después con todo lo demás estoy de acuerdo, son esas pequeñas cosas. Ni idea si esto te va a llegar por mail o se va a perder en github. Sino después te hablo por  whatsapp jaja.

Commit:
Administrador --> Manejo de Excepciones
Consulta --> Arreglo de Tipos
Usuario --> Eliminación de funciones duplicadas
Controlador:
	setRol
	ABM Usuarios
	ABM Clientes
	ABM Productos
	Creación de Reclamos
	Cierre de Reclamos
	Agregar Acción a reclamos
	testConnection
Prueba:
	Prueba ABM Clientes con Controlador

Commit:
Merge branch 'master' of
https://github.com/MacarenaBS/AplicacionesInteractivas into Eze

Conflicts:
	src/connections/ClientesDAO.java
	src/connections/UsuariosDAO.java
	src/controlador/Controlador.java
	src/usuarios/Administrador.java
